components/SiteNavigation.vue

```diff
@@ -97,7 +97,7 @@ const navigation = [
     children: [
       {
         title: 'Merchant Integrations',
-        path: '/guides/merchant-integrations',
+        path: '/guides/point-of-sale',
```

Comment:
This causes both items in the nav bar to be shown as active, non-blocking for me.
<img width="311" alt="Screenshot 2023-01-25 at 8 33 14 AM" src="https://user-images.githubusercontent.com/64108933/214390793-446e19b5-3cf7-42a1-a3a0-e3b60dcb2893.png">

Reply:
Yep - I've added a task to fix this later on